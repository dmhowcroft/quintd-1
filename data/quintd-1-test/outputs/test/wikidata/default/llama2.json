{
    "dataset": "wikidata",
    "model": "llama2",
    "setup": {
        "params": {
            "max_new_tokens": 512,
            "repetition_penalty": 1.0,
            "do_sample": false,
            "truncation_length": 32768,
            "auto_max_new_tokens": false,
            "mode": "instruct"
        },
        "prompt": "Based on the given data:\n```\n{DATA}\n```\n\nYour task is to write a brief, fluent, and coherent single-paragraph {output} in natural language. The text should be balanced and neutral. Make sure that all the facts mentioned in the text can be derived from the input data, do *not* add any extra information.",
        "start_with": "Sure! Here is the {output}:\n\"",
        "output": {
            "ice_hockey": "ice hockey game summary",
            "openweather": "five-day weather forecast",
            "gsmarena": "product description",
            "owid": "chart caption",
            "wikidata": "entity description"
        },
        "name": "default"
    },
    "generated": [
        {
            "in": "Based on the given data:\n```\nHundertwasser's Rainy Day\n---\n- cast member: Friedensreich Hundertwasser\n- composer: Arik Brauer\n- director: Peter Schamoni\n- director of photography: Josef Kaufmann\n- film editor: Heidi Genée\n- filming location: Venice\n- genre: documentary film\n- instance of: short film\n- nominated for: Academy Award for Best Documentary (Short Subject)\n- original language of film or TV show: German\n```\n\nYour task is to write a brief, fluent, and coherent single-paragraph entity description in natural language. The text should be balanced and neutral. Make sure that all the facts mentioned in the text can be derived from the input data, do *not* add any extra information.",
            "out": "Friedensreich Hundertwasser's Rainy Day is a 1966 documentary film directed by Peter Schamoni and produced by Arik Brauer. The film features the Austrian artist Friedensreich Hundertwasser and was shot in Venice, Italy. It was nominated for an Academy Award for Best Documentary (Short Subject) in 1967.",
            "tokens": {
                "in": 181,
                "out": 83
            }
        },
        {
            "in": "Based on the given data:\n```\nCatalan Republic\n---\n- basic form of government: republic\n- different from: Estat Català\n- driving side: right\n- head of government: Carles Puigdemont\n- instance of: state with limited recognition\n- instance of: historical country\n- located in or next to body of water: Mediterranean Sea\n- located in time zone: 00\n- located on terrain feature: Iberian Peninsula\n- public holiday: National Day of Catalonia\n```\n\nYour task is to write a brief, fluent, and coherent single-paragraph entity description in natural language. The text should be balanced and neutral. Make sure that all the facts mentioned in the text can be derived from the input data, do *not* add any extra information.",
            "out": "Catalan Republic is a state with limited recognition located in the Iberian Peninsula, bordered by the Mediterranean Sea. It has a right-hand traffic system and a head of government named Carles Puigdemont. Catalan Republic is an instance of a historical country and a state with limited recognition.",
            "tokens": {
                "in": 180,
                "out": 67
            }
        },
        {
            "in": "Based on the given data:\n```\nRatlam State\n---\n- described by source: Brockhaus and Efron Encyclopedic Dictionary\n- instance of: historical country\n- instance of: princely state of the British Raj\n- located in the administrative territorial entity: Central Provinces\n```\n\nYour task is to write a brief, fluent, and coherent single-paragraph entity description in natural language. The text should be balanced and neutral. Make sure that all the facts mentioned in the text can be derived from the input data, do *not* add any extra information.",
            "out": "Ratlam State was a princely state of the British Raj, located in the Central Provinces of India. It was ruled by the Sindhia family, who were granted the title of Maharaja in 1818. The state was known for its rich cultural heritage and was home to several temples and historical sites, including the Ratlam Fort, which was built in the 17th century.",
            "tokens": {
                "in": 128,
                "out": 89
            }
        }
    ]
}