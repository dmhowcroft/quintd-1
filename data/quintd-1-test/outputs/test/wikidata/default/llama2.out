Friedensreich Hundertwasser's Rainy Day is a 1966 documentary film directed by Peter Schamoni and produced by Arik Brauer. The film features the Austrian artist Friedensreich Hundertwasser and was shot in Venice, Italy. It was nominated for an Academy Award for Best Documentary (Short Subject) in 1967.
Catalan Republic is a state with limited recognition located in the Iberian Peninsula, bordered by the Mediterranean Sea. It has a right-hand traffic system and a head of government named Carles Puigdemont. Catalan Republic is an instance of a historical country and a state with limited recognition.
Ratlam State was a princely state of the British Raj, located in the Central Provinces of India. It was ruled by the Sindhia family, who were granted the title of Maharaja in 1818. The state was known for its rich cultural heritage and was home to several temples and historical sites, including the Ratlam Fort, which was built in the 17th century.
